geniza/corpus/templates/admin/corpus/document/change_form.html

```diff
@@ -18,11 +18,21 @@
         </fieldset>
     {% endif %}
     {% if original.has_transcription %}
-        <h2>Transcription</h2>
-        {% for footnote in original.footnotes.all %}
-            {% if footnote.content %}
-                {% include "footnotes/transcription.html" with transcription=footnote.content %}
-            {% endif %}
-        {% endfor %}
+        <fieldset class="module aligned transcriptions-field">
+            <div class="form-row">
+                <label>Transcription(s)</label>
+                {% regroup original.footnotes.all by source as source_list %}
```

Comment:
can't we just iterate over our `digital_editions` method? and I don't think we should regroup (don't need to, but also better to be explicit about what is attached where e.g. when transcription content needs to be consolidated in TEI after merging documents)

Reply:
Done in c62ff7a 👍